src/sentry/testutils/factories.py

```diff
@@ -899,10 +900,19 @@ def inject_performance_problems(jobs, _):
     @staticmethod
     @assume_test_silo_mode(SiloMode.REGION)
     def store_event(
-        data, project_id: int, assert_no_errors: bool = True, sent_at: datetime | None = None
+        data,
+        project_id: int,
+        assert_no_errors: bool = True,
+        event_type: str = "default",
+        sent_at: datetime | None = None,
     ) -> Event:
-        # Like `create_event`, but closer to how events are actually
-        # ingested. Prefer to use this method over `create_event`
+        """
+        Like `create_event`, but closer to how events are actually
+        ingested. Prefer to use this method over `create_event`
+        """
+        if event_type == "error":
```

Comment:
I'm open to other suggestions here - we have `create_performance_issue` and `store_functions` (which calls `store_event` and passes transactions) already, and you don't _typically_ need tests to have `error` type events, so I was also thinking of making a `create_error_issue` method.